csfieldguide/templates/interactives/training-ground.html

```diff
@@ -0,0 +1,114 @@
+{% extends interactive_mode_template %}
+
+{% load i18n %}
+{% load static %}
+
+{% block html %}
+  <div class="my-3 container-fluid text-center">
+    <div id="ai-warning" class="row">
+      {% trans 'Sorry, this interactive is only available on screens wider than 490px, and optimised for those wider than 610px.' %}
+    </div>
+    <div id="interactive-training-ground" class="container start-screen">
+      <h1>{% trans 'trAIning ground' %}</h1>
+      <br>
+      <h5>{% blocktrans %}In this game, take turns with the AI (Nathaniel) picking 1, 2, or 3 sticks.
+        Whoever picks the last stick wins!<br>
+        Watch how Nathaniel's picks change as it plays more games.{% endblocktrans %}
+      </h5>
+      <div class="col-12 my-3">
+        <div class="row black-background">
+          <div id="game-area" class="col-7">
+            <div class="row">
+              <div class="col-4">{% trans 'Sticks remaining:' %} <span id="remaining-sticks"></span></div>
+              <div class="col-4">{% trans 'AI wins:' %} <span id="ai-wins"></span></div>
+              <div class="col-4">{% trans 'Games played:' %} <span id="games-played"></span></div>
+            </div>
+            <div id="sticks-area" class="row text-left"></div>
+            <div id="status-text" class="col-12 text-center"></div>
+            <button type="button" class="btn btn-success m-2" id="button_start">{% trans 'START' %}</button>
```

Comment:
'Start' is also all in caps where as 'quit' is not.

Reply:
They all have different meanings. Granted they're similar enough so I'll change the three you've mentioned to regular case, but (I'm saying preemptively) STOP needs to stay caps because it's an emergency button